daml-script/test/BUILD.bazel

```
@@ -138,7 +138,10 @@ da_scala_library(
         "@maven//:org_scalaz_scalaz_core",
     ],
     scalacopts = lf_scalacopts_stricter,
-    tags = ["cpu:4"],
+    tags = [
+        "cpu:4",
+        "exclusive",
+    ],
```

```suggestion
    tags = [
        "cpu:4",
        "exclusive",
    ],
```
Why do we need the exclusive here ?

---

I was just trying things to see if I can make the canton setup stop timing out - it didn't work though :(
Do you have any idea why this might be happening?

---

The issue is that you use `override lazy val devMode = true` but your test is not marked  with `dev-canton-test` bazel flag.

---

OH I had no idea!
It was in dev mode so I could test some things and I completely forgot - you're a hero Remy

---

Never mind, I forgot i need dev mode because daml3 script runs only in LF 1.dev.
If adding the dev-canton-test will fix the problem, does that mean that dev canton is currently broken?

---

What do you need for 1.dev ?

---

Nothing specific, more than script is a dev feature, so I just put it on the dev LF version - it could probably run on LF1.15, I just didn't go through the trouble of generating daml3-script for a bunch of LF versions, given I don't even know if LF1.15 will be most recent by the time daml3-script is released

---

Fair enough.

---

once one #17157 is merged `canton-dev-test` should work again. 
Please revert the last 2 changes and mark your test with `dev-canton-test`. 
Just keep in mind that your tests will not be running when `SKIP_DEV_CANTON_TESTS=true`

---

Right okay thanks, I'll do that and wait until 17157 is merged so I can be sure the tests pass

---

Oh I see, my bad - i thought the squash and merge would hide that